### PR TITLE
Pass startup_time to telemetry

### PR DESCRIPTION
**Pull request by tomaka**
cc @maciejhirsz 

Passes the node startup time to the telemetry when connected.
This way, if we disconnect then reconnect, the telemetry knows that it's just the connection that closed and not the node as a whole.

### CODE REVIEW

File: core/service/src/builder.rs [hunk 1]
```diff
@@ -1095,6 +1098,9 @@ ServiceBuilder<
 				endpoints,
 				wasm_external_transport: config.telemetry_external_transport.take(),
 			});
+			let startup_time = SystemTime::UNIX_EPOCH.elapsed()
+				.map(|dur| dur.as_millis())
```
Comment (bkchr) [inline core/service/src/builder.rs]: Isn't seconds enough?

Comment (arkpar) [reply core/service/src/builder.rs]: This would pass machine start time, right? We want node (process) run time.

Comment (arkpar) [reply core/service/src/builder.rs]: Telemetry should show node running time regardless of when we connected to telemetry. E.g. if the node was started an hour ago, and it only connected to telemetry 15 minutes ago, it should still show "1 hour". For this to work we need to pass our real startup time I believe.

Comment (tomaka) [reply core/service/src/builder.rs]: This code is called only once, when we build the service.
`startup_time` is then cloned (well, copy'ed) when we connect to the telemetry.

Comment (arkpar) [reply core/service/src/builder.rs]: I misread it

Comment (tomaka) [reply core/service/src/builder.rs]: > Isn't seconds enough?

@maciejhirsz told me milliseconds were better.

**Pull request merged**